python-avd/pyavd/_eos_designs/structured_config/network_services/vlans.py

```diff
@@ -98,6 +98,20 @@ def _get_vlan_config(
             name=vlan.name,
             tenant=tenant.name,
         )
+        if vlan.address_locking.ipv4:
+            if self.inputs.address_locking_settings.dhcp_servers_ipv4 or self.inputs.address_locking_settings.locked_address.ipv4_enforcement_disabled:
+                vlans_vlan.address_locking.address_family.ipv4 = vlan.address_locking.ipv4
+            else:
+                msg = f"To configure address locking ipv4 for tenant: {tenant.name} and vlan id: {vlan.id} either \
+`address_locking_settings.dhcp_servers_ipv4` or `address_locking_settings.locked_address.ipv4_enforcement_disabled` is required."
+                raise AristaAvdInvalidInputsError(msg)
+        if vlan.address_locking.ipv6:
+            if self.inputs.address_locking_settings.dhcp_servers_ipv4 or self.inputs.address_locking_settings.locked_address.ipv6_enforcement_disabled:
+                vlans_vlan.address_locking.address_family.ipv6 = vlan.address_locking.ipv6
+            else:
+                msg = f"To configure address locking ipv6 for tenant: {tenant.name} and vlan id: {vlan.id} either \
+`address_locking_settings.dhcp_servers_ipv4` or `address_locking_settings.locked_address.ipv6_enforcement_disabled` is required."
```

Comment:
Please align the error message with what we have elsewhere:

```
  f"Unable to set EVPN RD/RT for vlan {vlan.id} in Tenant '{tenant.name}'"
```


```suggestion
                msg = (
                    f"To configure address locking ipv6 for vlan {vlan.id} in Tenant '{tenant.name}' either `address_locking_settings.dhcp_servers_ipv4` "
                    "or `address_locking_settings.locked_address.ipv6_enforcement_disabled` is required."
                )
```